src/components/interfaces/CMakeLists.txt

```
@@ -28,6 +28,15 @@
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
+# Copy RPC spec submodule files to interfaces directory
+if(EXISTS ${CMAKE_SOURCE_DIR}/tools/rpc_spec/)
+  execute_process(COMMAND cp ${CMAKE_SOURCE_DIR}/tools/rpc_spec/MOBILE_API.xml ${CMAKE_CURRENT_SOURCE_DIR}/)
+  execute_process(COMMAND cp ${CMAKE_SOURCE_DIR}/tools/rpc_spec/MOBILE_API.xsd ${CMAKE_CURRENT_SOURCE_DIR}/)
+else ()
+  message( FATAL_ERROR "Missing the RPC Spec submodule" )
+  message( FATAL_ERROR "Please run `git submodule init` in the SDL Core source directory" )
```

`submodule init` doesn't get the files from the submodule `submodule update` does that
```suggestion
  message( FATAL_ERROR "Please run `git submodule update --init` in the SDL Core source directory" )
```